crate2nix/templates/nix/crate2nix/default.nix

```diff
@@ -144,17 +144,15 @@ rec {
           done
```

Comment:
Awesome opportunity to check up on the comments -- do they make sense if we don't have all of the context in our brains anymore?

I cannot comment the function doc directly because it is not close enough to the change. It says "Returns a crate" where it probably meant to say "Returns a derivation"

It would also be nice to document "crate" vs "testCrate"